database.js

```diff
@@ -0,0 +1,9 @@
+const db = 'mongodb://localhost/lizardboard'
+const url = 'mongodb://127.0.0.1:27017/lizardboard';
+const MongoClient = require('mongodb').MongoClient, assert = require('assert');
+
+MongoClient.connect(url, (err, res, db) => {
```

Comment:
I think we need to make this a promise and export the creation of the promise as a function - otherwise we can't guarantee that we have successfully connected to the database when the server starts...